plugins/nessus.rb

```diff
@@ -1190,6 +1190,7 @@ def cmd_nessus_db_import(*args)
         if is_scan_complete(scan_id)
           print_status("Exporting scan ID #{scan_id} is Nessus format...")
           export = @n.scan_export(scan_id, 'nessus')
+          status = nil
```

Comment:
A safer option would be to initialize the variable with an empty hash (`{}`). This would prevent any call to `status['...']` to break if `status` is `nil`.